services/system/Accounts/plugin/accounts/src/interfaces/admin.rs

```diff
@@ -40,10 +49,19 @@ impl Admin for AccountsPlugin {
             TransactAuthApi::get_query_token(&Client::my_service_account(), &user).unwrap();
```

Comment:
Temporarily, as a sanity check to simplify auditing authenticated queries, I think you should add in a `println` here, so we see the retrieved token. 

This was previously done in the GUI code, but that part is now being deleted by this pr.

Reply:
as you said in Slack, I've done this in the next PR